### DISCUSSION TITLE
How to be a maintainer ?

### DISCUSSION THREAD

**Discussion by adsingh14** [simple-icons/simple-icons#6708] [General]
I want to maintain this repo. What are the prerequisites ?

**Comment by jorgeamadosoria** [comment]
you would need to be recommended by a current maintainer.
The fastest and more reliable way would be to make a few contributions so we can see your body of work, and you can decide if the work of creating the icons and maintaining the other repos is for you.
So... start by contributing a few of the 'good first issue' requests (or even some that are not labelled at such!) and we can go from there.

We are always interested in new maintainers, there is more than enough activities to go around!

**Reply by adsingh14** [reply]
Alright! I've helped 2 users who've not posted issues properly. That'd be counted too, though?

**Reply by jorgeamadosoria** [reply]
yes, all your activity is. 
but word to the wise: don't burn yourself trying to do too much too fast. It's better to emphasize quality, and a sustainable pace.
Be patient, and more importantly, take care of your local environment. It's not a race.

**Comment by adsingh14** [comment]
Is it mandatory to be consistent for months to be a maintainer?

I am asking casually.

**Reply by LitoMore** [reply]
No. We maintain this project in our free time.

**Reply by adsingh14** [reply]
That's what I do.
